bridge_adaptivity/module/models.py

```diff
@@ -149,7 +149,7 @@ def get_absolute_url(self):
         return reverse('module:course-detail', kwargs={'course_slug': self.slug})
 
     def __str__(self):
-        return self.name
+        return "<Course: {}>".format(self.name)
```

Comment:
because all other models have string representation like this and also to show what instance type it is, not only `name` which does not say anything about type of instance.